dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -417,4 +419,94 @@ public void testInvalidDockstoreYmlFiles() throws Exception {
             assertTrue("There should be 1 unsuccessful event", failEvents.stream().filter(lambdaEvent -> !lambdaEvent.isSuccess()).count() == 1);
         }
     }
+
+    /**
+     * This tests the GitHub release with .dockstore.yml located in /.github/.dockstore.yml
+     */
+    @Test
+    public void testGithubDirDockstoreYml() throws Exception {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
+        final ApiClient webClient = getWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        WorkflowsApi client = new WorkflowsApi(webClient);
+
+        client.handleGitHubRelease(githubFiltersRepo, BasicIT.USER_2_USERNAME, "refs/tags/1.0", installationId);
+        Workflow workflow = client.getWorkflowByPath("github.com/" + githubFiltersRepo + "/filternone", "", false);
+        assertNotNull(workflow);
+    }
+
+    /**
+     * This tests filters functionality in .dockstore.yml ; Workflow filters are configured as follows:
```

Comment:
Code base is getting large enough, please add a link to https://github.com/DockstoreTestUser2/dockstoreyml-github-filters-test in the comment here for reviewers (we haven't done this before)